content/changelog-stable/index.html

```diff
@@ -54,7 +54,76 @@
 
   <div id="rc" style="display:none;">
   </div>
-  
+
+<h3><a name=v2.19.1>What's new in 2.19.1</a> (2016-09-28)</h3>
+
+<div><strong>Changes from 2.19:</strong></div>
+
+<ul class=image>
+  <li class="bug">
+    Fixed the missing icon in the System Script console.
+    (<a href="https://issues.jenkins-ci.org/browse/JENKINS-37814">issue 37814</a>)
+  <li class="bug">
+    Fixed background color in the <code>ComboBoxList</code> element in order to make options visible.
+    (<a href="https://issues.jenkins-ci.org/browse/JENKINS-37549">issue 37549</a>)
+  <li class="bug">
+    Fixed editing default view description with automatic refresh.
+    System message is not being displayed instead of the view description.
+    (<a href="https://issues.jenkins-ci.org/browse/JENKINS-37360">issue 37360</a>)
+  <li class="bug">
+    Do not process <code>null</code> CRON specifications in build triggers.
+    (<a href="https://issues.jenkins-ci.org/browse/JENKINS-36748">issue 36748</a>, enhances fix in 2.15)
+  <li class="bug">
+    Setup wizard now checks if the restart is supported on the system before displaying the restart button.
+    (<a href="https://issues.jenkins-ci.org/browse/JENKINS-33374">issue 33374</a>)
+  <li class="bug">
+    Test Windows junctions before Java 7 symlink in symbolic link checks.
+    (<a href="https://issues.jenkins-ci.org/browse/JENKINS-29956">issue 29956</a>)
+</ul>
+
+<div><strong>Notable changes since 2.7.4:</strong></div>
+
+<ul class=image>
```

Comment:
Remoting upgrades are missing. 2.60 => 2.62. https://github.com/jenkinsci/remoting/blob/master/CHANGELOG.md . Maybe not so notable